rpm/harbour-wunderfitz.spec

```diff
@@ -42,13 +42,15 @@ Wunderfitz is a mobile dictionary application, supporting the Heinzelnisse dicti
 
 %build
 # >> build pre
+export APP_VERSION=%{version}
```

Comment:
We can use `printf "#ifndef APP_VERSION\n#define APP_VERSION \"%s\"\n#endif" "%{version}" > src/version.h` here instead of `export ...`.